src/pyskindose/calculate_dose/calculate_irradiation_event_result.py

```diff
@@ -1,18 +1,19 @@
 from typing import List, Dict, Any
-
+from tqdm import tqdm
 import numpy as np
 import pandas as pd
-from pyskindose import Phantom, constants as const
+import logging
+from pyskindose import Phantom, constants as c
 from pyskindose.calculate_dose.add_correction_and_event_dose_to_output import (
     add_corrections_and_event_dose_to_output,
 )
-
-# from pyskindose.calculate_dose.calculate_dose import logger
 from pyskindose.calculate_dose.perform_calculations_for_new_geometries import (
     perform_calculations_for_new_geometries,
 )
 from scipy.interpolate import CubicSpline
```

Comment:
Move to before imports of pyskindose modules

Reply:
corrected